arenadata/Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM rockylinux:8.8 as base
 
+ARG CC=clang
+ARG CXX=clang++
+
+RUN /bin/bash -c $' \n\
+	if [ -z "${CC}" ] || [ -z "${CXX}" ]; then \n\
+		echo "User must provide CC and CXX" \n\
+		exit 1 \n\
+	fi'
```

Comment:
Can we remove this step? The default values for CC and CXX are set. If user set CC and CXX to empty, then GCC is used.